crimson/verifybamid.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+    crimson.verifybamid
+    ~~~~~~~~~~~~~~
+
+    verifyBamId file parsing.
+
+    :copyright: (c) 2017 Dave Larson <delarson@wustl.edu>
+    :license: BSD
+
+"""
+import re
+import os
+
+import click
+
+from .utils import convert, get_handle
+
+
+__all__ = ["parse"]
+
+
+def parse(in_data):
+    """Parses verifyBamId output into a dictionary.
+
+    :param in_data: Input verifyBamId contents.
+    :type in_data: str or file handle
+    :returns: Parsed verifyBamId values.
+    :rtype: dict
```

Comment:
Looking at the code, the return type seems to either be a dictionary (when `contents` has one item) or a list of dictionaries (when `contents` has two or more items). Would you mind updating the `rtype` value if this is indeed the case?